### DISCUSSION TITLE
Installing scripts on Linux

### DISCUSSION THREAD

**Discussion by kopi23** [Stellarium/stellarium#2231] [Q&A]
Hi, after successfully testing my script in the Script Console I wanted to make it permanently available. I tried placing a myscript.ssc file with the contents in my /home/myuser/.stellarium/scripts directory. But after restarting Stellarium (0.21.3), I can't find it in the Scripts tab of the Configuration window. Should this be possible?
Thank you, kopi

**Comment by alex-w** [comment]
Please check extension of the file

**Reply by kopi23** [reply]
I'm quite sure that's correct as I placed the same file into the C:\Program Files\Stellarium\scripts directory on my Windows machine and there I can see and run it :(

**Reply by alex-w** [reply]
> I'm quite sure that's correct as I placed the same file into the C:\Program Files\Stellarium\scripts directory on my Windows machine and there I can see and run it :(

`/home/myuser/.stellarium/scripts` - this is path for Linux and UNIX

**Reply by kopi23** [reply]
On my Windows machine I placed it into C:\Program Files\Stellarium\scripts\ which works fine :)

On my Linux machine I placed it into /home/myuser/.stellarium/scripts/ since I don't know if I can copy anything inside a snap and it seems it doesn't get picked up :(
I checked that the snap starts with the same user. I even tried adding the execution mod to the myscript.ssc. So far no luck.

**Reply by gzotti** [reply]
Don't place a script in the program installation directory on Windows. There is a User Directory for that. 

You finally revealed you are using some snap package of Stellarium. These are not ours. Consider asking the package manager of your snap for assistance.

**Reply by kopi23** [reply]
Sorry for the confusion. I've posted my question over at https://github.com/t4saha/stellarium-snap/issues/2
Thank you, kopi

**Reply by t4saha** [reply]
@kopi23 Snap package uses separate xdg structure. all config & data files are stored inside `/home/myuser/snap/{App_Name}/common/`
Use this :  `/home/myuser/snap/stellarium-daily/common/.stellarium/scripts`

**Reply by kopi23** [reply]
@t4saha thank you very much over here too :)
